### PR TITLE
WPW-77: add config files

### PR DESCRIPTION
**Pull request by wberendt**
- The transactions data are in following config files now:
examples/pi-led/pi-led.json
examples/sample-consumer/sample-consumer.json
examples/sample-producer-callbacks/sample-producer-callbacks.json
examples/securenet-example/consumer/consumer.json
examples/securenet-example/producer/producer.json

### CODE REVIEW

File: examples/securenet-example/consumer/consumer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "firstName": "Joe",
+    "lastName": "Bloggs",
+    "expMonth": 12,
+    "expYear": 2020,
+    "cardNumber": "34343434343434",
+    "type": "Card",
+    "cvc": "123"
+  },
+  "pspConfig": {
+    "psp_name": "securenet",
```
Comment (maciej-wakula-mobica) [inline examples/securenet-example/consumer/consumer.json]: Indentation issue

File: examples/securenet-example/producer/producer.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "pspConfig": {
+    "psp_name": "securenet",
```
Comment (maciej-wakula-mobica) [inline examples/securenet-example/producer/producer.json]: Indentation issue

File: examples/pi-led/main.go [hunk 3]
```diff
@@ -24,6 +25,13 @@ const (
 
 func main() {
 
+	cfgFileName := "pi-led.json"
+	cfg, err := exutils.LoadConfiguration(cfgFileName)
+	if err != nil {
+		fmt.Println("error, failed to read config file", cfgFileName, ":", err)
+		os.Exit(1)
```
[the remark this answers is not in the document]
Comment (wberendt) [reply examples/pi-led/main.go]: I think fmt.Println() is ok in this case, the error is displayed directly after run, so it's useful.

File: examples/sample-consumer/main.go [hunk 4]
```diff
@@ -5,25 +5,24 @@ import (
 	"os"
 	"time"
 
-	log "github.com/sirupsen/logrus"
+	"github.com/WPTechInnovation/wpw-sdk-go/examples/exutils"
 	"github.com/WPTechInnovation/wpw-sdk-go/wpwithin"
 	"github.com/WPTechInnovation/wpw-sdk-go/wpwithin/psp"
 	"github.com/WPTechInnovation/wpw-sdk-go/wpwithin/psp/onlineworldpay"
-	"github.com/WPTechInnovation/wpw-sdk-go/wpwithin/types"
+	log "github.com/sirupsen/logrus"
```
[the remark this answers is not in the document]
Comment (wberendt) [reply examples/sample-consumer/main.go]: yes :)

File: examples/sample-consumer/main.go [hunk 5]
```diff
@@ -34,7 +33,14 @@ func main() {
 
 	initLog()
 
-	hceCard := initHCECard()
+	cfgFileName := "sample-consumer.json"
+	cfg, err := exutils.LoadConfiguration(cfgFileName)
+	if err != nil {
+		fmt.Println("error, failed to read config file", cfgFileName, ":", err)
```
[the remark this answers is not in the document]
Comment (wberendt) [reply examples/sample-consumer/main.go]: as above

File: examples/securenet-example/consumer/consumer.json (hunk 1 above)
Comment (wberendt) [reply examples/securenet-example/consumer/consumer.json]: ok, I'll correct it.

File: examples/securenet-example/producer/producer.json (hunk 2 above)
Comment (wberendt) [reply examples/securenet-example/producer/producer.json]: ok

**Pull request merged**